### PR TITLE
Sex & Gender Guidance: Fixes for FHIR-46415, FHIR-46429, FHIR-46430, FHIR-46428

### PR DESCRIPTION
**Pull request by dt-r**
This PR fixes:
- [FHIR-46415](https://jira.hl7.org/browse/FHIR-46415) AU Base: Sex and gender guidance
- [FHIR-46429](https://jira.hl7.org/browse/FHIR-46429) The guidance for Name to Use (NtU) from AU Core should be migrated to AU Base
- [FHIR-46430](https://jira.hl7.org/browse/FHIR-46430) The guidance for exchanging Sex Assigned at Birth from AU Core should be migrated to AU Base
- The guidance part of [FHIR-46428](https://jira.hl7.org/browse/FHIR-46428) Recorded Sex or Gender type should not be populated with source values

Includes updates to:
- changes.md
- index.md
- examples (incl Australianising the sex and gender examples)
- profile-specific implementation guidance
- new guidance page
- menu.xml

### CODE REVIEW

File: input/pagecontent/sexgender.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+
+Gender Identity is represented with the [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) extension. In AU Base the value element of the [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) extension is constrained to be [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)).
+
+When populating the value element of the [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) extension:
+  - *Man or male* may be represented by sending the SNOMED CT code 446151000124109\|Identifies as male gender\|
+  - *Woman or female* may be represented by sending the SNOMED CT code 446141000124107\|Identifies as female gender\|
+  - *Non-binary* may be represented by sending the SNOMED CT code 33791000087105\|Identifies as nonbinary gender\|
+  - *[I/They] use a different term (please specify)*  may be represented by sending only `text` and no `coding`
+  - *Prefer not to answer* may be represented by sending the code "asked-declined"
```
Comment (JayMurdoch) [inline input/pagecontent/sexgender.md:119]: Suggest: ... sending the **Data Absent Reason** code “asked-declined”

File: input/pagecontent/sexgender.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+Pronouns is represented with the [Individual Pronouns](http://hl7.org/fhir/StructureDefinition/individual-pronouns) extension. In AU Base the value element of the [Individual Pronouns](http://hl7.org/fhir/StructureDefinition/individual-pronouns) extension is constrained to be [Australian Pronouns](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/australian-pronouns-1) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)).
+
+When populating the value element of the [Individual Pronouns](http://hl7.org/fhir/StructureDefinition/individual-pronouns) extension:
+- *they/them/their/theirs/themselves* may be represented by sending the LOINC Answer (LA) code LA29520-6
+- *she/her/her/hers/herself* may be represented by sending the LOINC Answer (LA) code LA29519-8
+- *he/him/his/his/himself* may be represented by sending the LOINC Answer (LA) code LA29518-0
+- *Prefer not to answer* may be represented by sending the Data Absent Reason code "asked-declined"
+- *Asked but not known* may be represented by sending the Data Absent Reason code "asked-unknown"
+- *Not stated or inadequately described* may be represented by the Data Absent Reason code "unknown"
```
Comment (JayMurdoch) [inline input/pagecontent/sexgender.md:44]: Suggest: ... represented by **sending** the ...

File: input/pagecontent/sexgender.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+
+When populating the value element of the [Individual Gender Identity](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) extension:
+  - *Man or male* may be represented by sending the SNOMED CT code 446151000124109\|Identifies as male gender\|
+  - *Woman or female* may be represented by sending the SNOMED CT code 446141000124107\|Identifies as female gender\|
+  - *Non-binary* may be represented by sending the SNOMED CT code 33791000087105\|Identifies as nonbinary gender\|
+  - *[I/They] use a different term (please specify)*  may be represented by sending only `text` and no `coding`
+  - *Prefer not to answer* may be represented by sending the code "asked-declined"
+  - *Not stated or inadequately described* may be represented by the code "unknown"
```
Comment (JayMurdoch) [inline input/pagecontent/sexgender.md:120]: Suggest: ... represented by **sending** the **Data Absent Reason** code "unknown"

File: input/pagecontent/sexgender.md [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+  - Where the workflow does not support obtaining a gender identity value, it may be represented by sending the code "not-asked"
```
Comment (JayMurdoch) [inline input/pagecontent/sexgender.md:121]: Suggest: ... represented by sending the **Data Absent Reason** code "not-asked"

File: input/pagecontent/sexgender.md [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+- [AU Base Related Person](StructureDefinition-au-relatedperson.html)
+
+Sex Assigned at Birth is represented with the [Person Recorded Sex or Gender](https://hl7.org/fhir/extensions/StructureDefinition-individual-recordedSexOrGender.html) extension:
+- the type element is populated with 1515311000168102\|Biological sex at birth\|
+- the value element is constrained to be [Biological Sex](https://healthterminologies.gov.au/fhir/ValueSet/biological-sex-1) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible))
+
+When populating the value element of the [Person Recorded Sex or Gender](https://hl7.org/fhir/extensions/StructureDefinition-individual-recordedSexOrGender.html) extension for Sex Assigned at Birth:
+  - *Male* may be represented by sending the SNOMED CT code 248153007\|Male\|
+  - *Female* may be represented by sending the SNOMED CT code 248152002\|Female\|
+  - *Intersex* may be represented by sending the SNOMED CT code 33791000087105\|Intersex\|
```
Comment (JayMurdoch) [inline input/pagecontent/sexgender.md:248]: Incorrect code for Intersex - change to 32570691000036108

File: input/pagecontent/sexgender.md [hunk 6]
```diff
@@ -0,0 +1,336 @@
...
+      "url" : "http://hl7.org/fhir/StructureDefinition/individual-genderIdentity",
+      "extension" : [
+        {
+          "url" : "value",
+          "valueCodeableConcept" : {
+            "coding" : [
+              {
+                "system" : "http://snomed.info/sct",
+                "code" : "446141000124107",
+                "display" : "Identifies as female gender (finding)"
```
Comment (JayMurdoch) [inline input/pagecontent/sexgender.md:139]: While not incorrect, typically the Preferred Term is used for display. Consider use of "Identifies as female gender" (ie without semantic tag).

File: input/pagecontent/sexgender.md [hunk 7]
```diff
@@ -0,0 +1,336 @@
...
+
+
+### Sex Assigned at Birth
+AU Base supports representation and exchange of the Sex Assigned at Birth data element (as defined in the [HL7 Cross Paradigm Implementation Guide: Gender Harmony - Sex and Gender Representation, Edition 1](https://hl7.org/xprod/ig/uv/gender-harmony/informative1/)) in:
+- [AU Base Patient](StructureDefinition-au-patient.html)
+- [AU Base Practitioner](StructureDefinition-au-practitioner.html)
+- [AU Base Related Person](StructureDefinition-au-relatedperson.html)
+
+Sex Assigned at Birth is represented with the [Person Recorded Sex or Gender](https://hl7.org/fhir/extensions/StructureDefinition-individual-recordedSexOrGender.html) extension:
+- the type element is populated with 1515311000168102\|Biological sex at birth\|
```
Comment (JayMurdoch) [inline input/pagecontent/sexgender.md:242]: Suggest: ... populated with **SNOMED CT code** 1515311000168102...

File: input/pagecontent/sexgender.md (hunk 6 above)
Comment (dt-r) [reply input/pagecontent/sexgender.md:139]: Not a required change. Some improvements can be made at later dates

File: input/pagecontent/sexgender.md (hunk 7 above)
Comment (dt-r) [reply input/pagecontent/sexgender.md:242]: I will for now, and then later we can rip them all out. I think this style is poor and am pleased we don't do it in AU Core.

File: input/pagecontent/sexgender.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/sexgender.md:120]: fixed

File: input/pagecontent/sexgender.md (hunk 4 above)
Comment (dt-r) [reply input/pagecontent/sexgender.md:121]: fixed

File: input/pagecontent/sexgender.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/sexgender.md:44]: fixed

File: input/pagecontent/sexgender.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/sexgender.md:119]: fixed

File: input/pagecontent/sexgender.md (hunk 5 above)
Comment (dt-r) [reply input/pagecontent/sexgender.md:248]: fixed

File: input/pagecontent/sexgender.md (hunk 1 above)
Comment (dbojicic) [reply input/pagecontent/sexgender.md]: Confirmed fix

File: input/pagecontent/sexgender.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/sexgender.md]: Confirmed

File: input/pagecontent/sexgender.md (hunk 3 above)
Comment (dbojicic) [reply input/pagecontent/sexgender.md]: Confirmed fix.

File: input/pagecontent/sexgender.md (hunk 4 above)
Comment (dbojicic) [reply input/pagecontent/sexgender.md]: Confirmed fix

File: input/pagecontent/sexgender.md (hunk 5 above)
Comment (dbojicic) [reply input/pagecontent/sexgender.md]: Confirmed fix

File: input/pagecontent/sexgender.md (hunk 6 above)
Comment (dbojicic) [reply input/pagecontent/sexgender.md:139]: Noted

File: input/pagecontent/sexgender.md (hunk 7 above)
Comment (dbojicic) [reply input/pagecontent/sexgender.md]: Confirmed

**Review by dbojicic — APPROVED** [review]

**Pull request merged**